src/types.ts

```diff
@@ -11,6 +11,28 @@ export type CustomInjectManifestOptions = InjectManifestOptions & {
   vitePlugins?: InjectManifestVitePlugins
 }
 
+export interface SvelteKitVitePluginOptions {
+  /**
+   * Timeout in miliseconds after SvelteKit prerender finish.
+   *
+   * When using prerender, SvelteKit Plugin will call `process.exit(0)`, configure a default timeout to allow the prerender to flush the result to disk.
+   *
+   * If not configured, the PWA Plugin will generate the PWA stuff without sleep, the result can be weird (empty service worker precache manifest).
+   *
+   * [Check this issue for more info](https://github.com/sveltejs/kit/issues/5306)
+   */
+  prerenderTimeout?: number
+
+  /**
+   * Use this option when you need to use a script after SvelteKit build.
+   *
+   * Check the `sveltekit-pwa` in the examples folder.
```

Comment:
I don't see any example where this is being set

Reply:
Maybe we can remove it, since the external pwa module is using just a subset of Vite plugins configured and so there is no need, the SvelteKit Vite Plugin will not be in the list.